packages/cli/__tests__/zosfiles/__unit__/delete/zfs/zfs.handler.unit.test.ts

```diff
@@ -17,38 +26,45 @@ describe("ZfsHandler", () => {
     });
 
     let deleteZfsSpy: any;
-    let processWithSessionSpy: any;
+    // let processWithSessionSpy: any;
 
     beforeEach(() => {
-        deleteZfsSpy= jest.spyOn(Delete, "zfs");
+        jest.spyOn(ConnectionPropsForSessCfg, "addPropsOrPrompt").mockResolvedValue({
+            hostname: "example.com"
+        });
+        deleteZfsSpy = jest.spyOn(Delete, "zfs");
         deleteZfsSpy.mockClear();
         deleteZfsSpy.mockImplementation(async () => defaultReturn);
     });
 
     it("should return success: true when --quiet (-fq) flag is used and file is not found", async () => {
-        // deleteZfsSpy.mockImplementationOnce(() => {
-        //     throw fileNotFoundError;
-        // });
-        const handler = new ZfsHandler();
-
-        processWithSessionSpy = jest.spyOn(handler, "processWithSession");
-        processWithSessionSpy.mockImplementation(() => {
+        deleteZfsSpy.mockImplementation(() => {
             throw fileNotFoundError;
         });
 
+        const handler = new ZfsHandler();
         const commandParameters: any = {
             arguments: {
                 fileSystemName: "ABCD",
+                forSure: true,
                 quiet: true,
-                forSure: true
+            },
+            response: {
+                progress: { endBar: jest.fn() },
+                data: { setObj: jest.fn() }
             }
         };
 
-        await expect(handler.process(commandParameters)).resolves.toEqual({ success: true });
+        // const error = new ImperativeError({msg: "IDC3012I ENTRY HLQ.MYNEW.ZFS NOT FOUND"});
+        // await expect(handler.processWithSession(commandParameters, {} as any)).rejects.toThrow(ImperativeError);
+        // await expect(handler.processWithSession(commandParameters, {} as any)).rejects.toThrow(error);
+        // await expect(handler.processWithSession(commandParameters, {} as any)).resolves.toEqual({ success: true });
+        await expect(handler.process(commandParameters)).resolves.toBe(undefined);
+        // await expect(handler.process(commandParameters)).resolves.toEqual({ success: true });
     });
 
     it("should throw file not found error (404) when --quiet is not used (-f)", async () => {
-        deleteZfsSpy.mockImplementationOnce(() => {
+        deleteZfsSpy.mockImplementation(() => {
```

Comment:
Hey @ATorrise,
I forgot to add the `mockImplementationOnce` back on both tests.
It might be the cause of other tests failing. Sorry about that 😢